shared/engine/index.js

```diff
@@ -427,8 +437,8 @@ class Engine {
     this._failOnError = true
   }
 
-  // Register a named callback when we disconnect from the server. Call if we're already disconnected
-  listenOnDisconnect(key: string, f: () => void) {
+  // Register a named callback when we disconnect from the server. Call if we're already disconnected. Callback should produce an action
```

Comment:
does `server` mean `service` here?

Reply:
those are kind used interchangably, we never talk directly to the api server so in our case server = service